management/src/main/java/io/micronaut/management/endpoint/EndpointDefaultConfiguration.java

```diff
@@ -105,6 +124,16 @@ public void setPath(String path) {
         }
     }
 
+    /**
+     * The endpoints context path. It must include a leading and trailing '/'. Default value ({@value #DEFAULT_ENDPOINT_CONTEXT_PATH}).
+     * @param contextPath The Context Path
+     */
+    public void setContextPath(String contextPath) {
```

Comment:
since contextPath must start with `/` and end with `/`, validate it here and throw a `ConfigurationException` if invalid.

Reply:
In fact, the comment is wrong, there is no need for it, I even believe that the comment about the context path for standard endpoints is also wrong.